src/runtime-rs/crates/runtimes/src/direct_volume/volume_ops.rs

```diff
@@ -0,0 +1,154 @@
+use anyhow::{anyhow, Ok, Result};
```

Comment:
I think it's better to put everything in this file into `src/tools/kata-ctl/src/ops/volume_ops.rs` cause it is implementing the subcommand of `kata-ctl`

Reply:
Thanks for the advice! Moving things to client side would make much sense. However, I did move `ResizeRequest` to `runtimes/src/shimmgmt/handlers.rs` and make some of the constants & mods public as this would avoid cyclic dependency issues and I think it's best if these are kept in server side.